packages/backend/src/api/request.ts

```diff
@@ -88,7 +88,7 @@ export function buildRequest(options: BuildRequestOptions) {
     let res: Response | undefined;
     try {
       if (formData) {
-        res = await runtime.fetch(finalUrl.href, {
+        res = await (process.env.NODE_ENV === 'test' ? fetch : runtime.fetch)(finalUrl.href, {
```

Comment:
Will need to think about a more ergonomic way of accomplishing this. With the way `fetch` is handled in the package `msw` can't intercept requests without it for now

Reply:
@BRKalow Any thoughts on how you would like to move forward here?

Reply:
@jacekradko can we try to consolidate into the runtime module? so at least usage of runtime.fetch doesn't need to be aware of the `NODE_ENV`